Parser.h

```diff
@@ -0,0 +1,11 @@
+#include <map>
+#include <regex>
+#include <iostream>
+#include <fstream>
+
+class Parser
+{
+public:
+    const std::map <std::string, std::string> parseJson(std::string json);
+    const std::map <std::string, std::string> parseJson(std::istream& jsonFile);
```

Comment:
Első ránézésre ezek akár statikusak is lehetnének.

Reply:
Átírtam őket statikusra.